lighthouse-core/gather/drivers/driver.js

```diff
@@ -487,8 +488,8 @@ class Driver {
     ].join(',');
 
     return this.sendCommand('Storage.clearDataForOrigin', {
-      origin,
-      storageTypes,
+      origin: origin,
```

Comment:
@paulirish had a point for this specific instance. Not sure if I completely agree, so he can explain if he wants :)